src/cryptography/x509/name.py

```diff
@@ -118,15 +118,21 @@ def oid(self) -> ObjectIdentifier:
     def value(self) -> str:
         return self._value
 
+    @property
+    def rfc4514_attribute_name(self) -> str:
+        """
+        The short attribute name (for example "CN") if available, otherwise the OID dotted string.
+        """
+        return _NAMEOID_TO_NAME.get(self.oid, self.oid.dotted_string)
+
     def rfc4514_string(self) -> str:
         """
         Format as RFC4514 Distinguished Name string.
 
         Use short attribute name if available, otherwise fall back to OID
         dotted string.
         """
-        key = _NAMEOID_TO_NAME.get(self.oid, self.oid.dotted_string)
-        return "%s=%s" % (key, _escape_dn_value(self.value))
+        return "%s=%s" % (self.rfc4514_attribute_name, _escape_dn_value(self.value))
```

Comment:
This will also need to be line-wrapped. (Running black should do it automatically!)

Reply:
Fixed, thanks! And sorry for the repeated turnarounds. Seems like I really need to get used to more zealous CI checks :)